nemo/collections/asr/data/audio_to_diar_label.py

```diff
@@ -274,7 +259,7 @@ def assign_labels_to_longer_segs(self, uniq_id, base_scale_clus_label):
         per_scale_clus_label = torch.tensor(per_scale_clus_label)
         return per_scale_clus_label, uniq_scale_mapping
 
-    def get_diar_target_labels(self, uniq_id, fr_level_target):
+    def get_diar_target_labels(self, uniq_id, sample, fr_level_target):
```

Comment:
add doc strings for sample

Reply:
possible to initialize with default values for these variables? comes in handy for debugging issues in the future

Reply:
This needs discussion. I can't fully understand what variable should be initilaized with default values.